corehq/blobs/tests/test_export.py

```diff
@@ -1,123 +1,90 @@
 import doctest
-import os
 import tarfile
-import uuid
 from io import BytesIO, RawIOBase
 from tempfile import NamedTemporaryFile
 
 from django.test import SimpleTestCase, TestCase
 
-from corehq.apps.hqmedia.models import (
-    CommCareAudio,
-    CommCareImage,
-    CommCareVideo,
-)
 from corehq.blobs import CODES, get_blob_db
-from corehq.blobs.export import EXPORTERS
+from corehq.blobs.export import BlobExporter
+from corehq.blobs.management.commands.run_blob_import import Command as ImportCommand
+from corehq.blobs.tests.fixtures import blob_db
 from corehq.blobs.tests.util import TemporaryFilesystemBlobDB, new_meta
 
 
-class TestBlobExport(TestCase):
-    @classmethod
-    def setUpClass(cls):
-        super().setUpClass()
+class TestBlobExporter(TestCase):
 
-        cls.db = TemporaryFilesystemBlobDB()
-        assert get_blob_db() is cls.db, (get_blob_db(), cls.db)
-        cls.data = data = b'binary data not valid utf-8 \xe4\x94'
-        cls.blob_metas = []
-        cls.not_found = set()
+    def test_only_blob_in_targeted_domain_is_exported(self):
+        # create blob that should be ignored
+        self.db.put(BytesIO(self.blob_data), meta=new_meta(domain='random', type_code=CODES.form_xml))
+        expected_meta = self.db.put(
+            BytesIO(self.blob_data), meta=new_meta(domain=self.domain, type_code=CODES.form_xml)
+        )
 
-        cls.domain_name = str(uuid.uuid4)
+        with NamedTemporaryFile() as out:
+            self.exporter.migrate(out.name, force=True)
+            with tarfile.open(out.name, 'r:gz') as tgzfile:
+                self.assertCountEqual({expected_meta.key}, tgzfile.getnames())
```

Comment:
Would `assertEqual` work here? It seems like it might.

The same applies below for other uses of `assertCountEqual`.

Reply:
Agreed with the exception of the test that tests multiple blobs. Does that seem fair?
```
self.assertCountEqual({form.key, multimedia.key}, tgzfile.getnames())
```